custom/icds_reports/static/js/directives/download/download.directive.js

```diff
@@ -414,8 +426,8 @@ function DownloadController($rootScope, $location, locationHierarchy, locationsS
         }
     };
 
-    vm.setMonthToPreviousIfBeforeThe15th = function (date) {
-        var offset = date.getDate() < 15 ? 1 : 0;
+    vm.setMonthToPreviousIfAfterThe15thAndTwoMonthsIfBefore15th = function (date) {
```

Comment:
Hey, What is this doing actually?. I think there has to be no change in the how performance report works. It never shows the filter for current month.

Reply:
we have to show before last month report if present day is first 15 days of the month or else we need to show last month report